ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -525,45 +525,6 @@ tests:
           cpu: 100m
           memory: 200Mi
     test:
-    - as: addon-ocs-consumer-tests
```

Comment:
@alfonsomthd  The ocs-consumer job was not removed after running `make jobs` after removing this job config.
Do I need to do it manually?

Reply:
Then yes, delete it manually. You can also run `make update` (that also includes `make jobs`) to verify that all is correct or at least `make ci-operator-config` (also included in `make update`).

Reply:
Had to remove it manually, there were no changes after running `make update`.
Thanks